scripts/build_package.py

```diff
@@ -251,7 +251,7 @@ def handle_external_repository(dep, clean, env, verbose):
         if source:
             tmp = tempfile.mkdtemp()
             LOG.debug("Downloading font files " + source + " to " + tmp)
-            for fformat, agent in font_user_agents.iteritems():
+            for fformat, agent in font_user_agents.items():
```

Comment:
We made similar changes other places in the code previously. The other thing that this is only in the packaging script.